app/jobs/send_message_job.rb

```diff
@@ -9,6 +9,8 @@ def perform(recipient_id)
       return
     end
 
-    MessageService.new.send_message(recipient, recipient.message_batch.message_template.body)
+    message_body = recipient.message_batch.message_template.body_backend.read(recipient.preferred_language)
```

Comment:
No strong preference, but would `Mobility.with_locale(recipient.preferred_language) { recipient.message_batch.message_template.body }` work here? Feels slightly more clear to me.

Reply:
I was debating between the 2 and wondering whether an explicit `set_body(locale)` method on `MessageTemplate` would be better.